src/handlers/motions/motionCreated/motionCreated.ts

```diff
@@ -24,6 +25,11 @@ export default async (event: ContractEvent): Promise<void> => {
         await writeMintTokensMotionToDB(event, parsedAction);
         break;
       }
+      case ColonyOperations.AddDomain:
+      case ColonyOperations.EditDomain: {
+        await writeManageDomainMotionToDB(event, parsedAction);
+        break;
+      }
```

Comment:
Will this overwrite the domain metadata entry in the case of a domain edit ?

Reply:
It shouldn't, this function's only purpose is to create a new `ColonyAction` entry